xds/src/main/java/io/grpc/xds/XdsNameResolver.java

```diff
@@ -876,7 +878,8 @@ public void onResourceDoesNotExist(final String resourceName) {
         if (RouteDiscoveryState.this != routeDiscoveryState) {
           return;
         }
-        String error = "RDS resource does not exist: " + resourceName;
+        String error = "RDS resource does not exist: " + resourceName + " xDS node ID: "
```

Comment:
This will have two node IDs included, one here and one in `cleanUpRoutes()`.

Reply:
Removed xDS node ID from here.